source/_integrations/workday.markdown

```diff
@@ -66,7 +66,7 @@ add_holidays:
   required: false
   type: list
 remove_holidays:
-  description: "Remove holidays (treat holiday as workday). Needs to formatted as `YYYY-MM-DD`."
+  description: "Remove holidays (treat holiday as workday). Can be formatted as `YYYY-MM-DD` or by name for a partial string match (e.g. Thanksgiving)."
```

Comment:
We should point users to where to find the correctly formatted strings to use. There is no nicely maintained list so I think the best compromise is to point to [the source](https://github.com/dr-prodigy/python-holidays/tree/master/holidays/countries) where they can look in the `.py` file for their country.

Reply:
I don't agree on sending our end-users to Python source files.

Reply:
@French what do you suggest as an alternate. `Holidays` is super picky on format and the library doesn't offer a clear list anywhere I could find

Reply:
Although I note that for example King's Day in the Netherlands is not translated in the Wikipedia page (presumably is in 
the Dutch version) but is in `Holidays`

Reply:
I have to agree that directing users to Python source files isn't the most elegant.  I'd thought the partial string match would be forgiving enough for the majority of cases and debug logs would be available for the rest, but a more accessible reference would be ideal.

I'd planned a future PR (once I got the hang of contributing) to add additional attributes to the workday integration, potentially including the list of "known holidays".  Would this be better?

Reply:
That sounds good although not sure attributes is the way to do it. Migrating to a config flow and allowing a picker in the UI seems like the best solution long term. Looking back I think the partial matching is going to work well enough and this is already more robust than date-format removal